playbooks/slurm-cluster.yml

```diff
@@ -23,6 +23,14 @@
 # Install software
 - include: software.yml
 
+# Set up NFS filesystem
+- include: nfs-server.yml hostlist=slurm-master
```

Comment:
might want to use the hostlist 'hack' here in case someone wants to set up the NFS server on a different system

Reply:
On reflection, I actually think the right way to do this is in the inventory file. I just pushed an update to refactor for new inventory groups.